DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -1289,5 +1289,37 @@ public void Get_section_content_should_not_have_otherSectionsExist_when_other_se
                 result!.OtherSectionsExist.Should().BeFalse();
             }
         }
+
+        [Test]
+        public void Get_section_content_tutorials_should_have_current_score()
+        {
+            // Given
+            const int customisationId = 19262;
+            const int candidateId = 2;
+            const int sectionId = 1010;
+            int[] expectedCurrentScores = { 5, 5, 2, 2, 5 };
+
+            // When
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            // Then
+            result.Tutorials.Select(tutorial => tutorial.CurrentScore).Should().Equal(expectedCurrentScores);
+        }
+
+        [Test]
+        public void Get_section_content_tutorials_should_have_possible_score()
+        {
+            // Given
+            const int customisationId = 19262;
+            const int candidateId = 2;
+            const int sectionId = 1010;
+            int[] expectedPossibleScores = { 5, 5, 2, 2, 5 };
```

Comment:
Should these really be the same as the current scores? If so could we please use a section which has different current scores to possible scores, so the tests are a bit more robust? 🙂